umcontroller/umcontroller.go

```diff
@@ -119,6 +128,11 @@ type CertificateProvider interface {
 	GetCertificate(certType string, issuer []byte, serial string) (certURL, keyURL string, err error)
 }
 
+// Decrypter decrypt and validate image provider interface.
```

Comment:
unclear comment: interface to decrypt and validate image?

Reply:
Done